test/bemjson-mix-test.js

```diff
@@ -238,4 +238,48 @@ describe('BEMJSON mix', function() {
         'b__e_modname_modval"></div></div>');
     });
   });
+
+  describe('should support nested mixes', function() {
```

Comment:
`describe('nested mixes', function() {`

`it('should support nested mix', function() {`

`it('should support nested mix with js params', function() {`